skimage/filters/__init__.pyi

```diff
@@ -11,6 +11,7 @@ __all__ = [
     "farid",
     "farid_h",
     "farid_v",
+    "filter_forward",
```

Comment:
This function wasn't part of our public API (at least not explicitly) and isn't used anywhere in our repo. Should we remove it? Otherwise we should really add some tests for it!